src/estimators/tests/bloom_filters_test.py

```diff
@@ -120,6 +120,12 @@ def test_multi_insertion_same_element(self):
     b1.add(1)
     self.assertEqual(UnionEstimator.estimate_cardinality(b1), 1)
 
+  def test_raise_error_with_full_bloom_filter(self):
+    b1 = BloomFilter(length=2, random_seed=3)
+    b1.add_ids(range(10))
+    with self.assertRaises(ValueError):
+      _ = UnionEstimator.estimate_cardinality(b1)
```

Comment:
This is confusing syntax. Why are we assigning a throwaway to the return of this function? Shouldn't we just call the function instead without assigning anything?

Reply:
another good call, will remove